src/utils/index.js

```diff
@@ -35,6 +35,24 @@ module.exports.isMobile = function () {
 };
 
 /**
+ * Returns tick throttle function that only fires callback after minimum interval.
+ * Note, this does not try to make up for any lost time by firing multiple callbacks.
+ * @param {callback}   The callback to fire; function (time, deltaTime) {...}
```

Comment:
`function` would be a reserved word clash... maybe `functionToThrottle`
`number` is less descriptive than `interval`, might be better to call `minimumInterval`